src/searchForm/autoComplete/overlay/index.tsx

```diff
@@ -12,27 +12,30 @@ const StyledAutoComplete = styled(autoComplete)`
     border: 1px solid ${color.border};
     border-radius: ${radius.l};
     padding: ${space.xl};
+    padding: ${space.xl};
+    padding-top: ${space.s};
     /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
     box-shadow: 0 1pt 4pt rgba(0, 0, 0, 0.16), 0 2pt 8pt rgba(0, 0, 0, 0.08);
+    background: ${color.defaultBackground};
 
     .kirk-textField-wrapper {
```

Comment:
Loud thinking: I found this much harder to review since we used the CSS nesting :/ (I need to expand all the above code and can't scroll quickly)

Reply:
Good point! I didn't thought about this kind of issues.

Reply:
I agree :/